x/ccv/provider/keeper/partial_set_security.go

```diff
@@ -61,3 +61,21 @@ func (k Keeper) HandleOptOut(ctx sdk.Context, chainID string, providerAddr types
 
 	return nil
 }
+
+func (k Keeper) HandleSetConsumerCommissionRate(ctx sdk.Context, chainID string, providerAddr types.ProviderConsAddress, commissionRate sdk.Dec) error {
```

Comment:
Add docstring. Also, is this the right file to put this in, especially as it's not related to PSS?

Reply:
Good point. Documented and moved to `distribution.go`.